scripts/main.js

```diff
@@ -213,7 +213,7 @@ async function createStaveSpellcastingEntry(stave, actor, existingEntry = null)
         const match = description.match(regex);
         if (!match) continue;
 
-        const strs = match[0].match(/(@UUID\[Compendium\.|@Compendium\[)(.*?)].*?}/g);
+        const strs = match[0].match(/(@UUID\[Compendium\.|@Compendium\[)(.*?)]({.*?})?/g);
```

Comment:
This makes the {} portion optional by placing it in a group and making that whole group optional.

Reply:
@jessev14 Can you help take a look? This fixed issues for me locally. 😄